roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh

```diff
@@ -80,6 +80,10 @@ EOF
       NEEDS_RESTART=1
     fi
 
+    if ! `systemctl -q is-active dnsmasq.service`; then
```

Comment:
Looks like it was added in systemd v198 prior to v209 (RHEL 7.1) and has defined rules regarding exit codes; 0 only if the service is active.